integration_test/star_mail_test.dart

```diff
@@ -1,11 +1,13 @@
+import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
+import 'package:gallery/keys.dart';
 import 'package:patrol/patrol.dart';
+
 import 'src/common_methods.dart';
-import 'package:gallery/keys.dart';
 
 void main() {
   patrolTest('Create new mail', nativeAutomation: true, ($) async {
-    await startAppTest($);
+    await startApp($);
     await navigateToEmail($);
     await $(K.email).tap();
     final deletedSubject =
```

Comment:
Change a name for more descriptive - we're not deleting anything in this scenario

Reply:
Still waiting for fix

Reply:
This comment concerns the line it is attached to - though it's good you corrected the test name as well, I missed it. The variable name `deletedSubject` is still waiting for better name though